framework/src/outputs/TableOutput.C

```diff
@@ -34,6 +34,9 @@ InputParameters validParams<TableOutput>()
   InputParameters params = validParams<AdvancedOutput<FileOutput> >();
   params += AdvancedOutput<FileOutput>::enableOutputTypes("postprocessor scalar vector_postprocessor");
 
+  // Option for writing vector_postprocessor time file
+  params.addParam<bool>("time_data", false, "When true and VecptorPostprocessor data exists, write a csv file containing the timestep and time information.");
```

Comment:
@permcody You do have something slightly wrong... the current VectorPostprocessor CSV files are only one _file_ per timestep (you said "each vector postprocessor writes it's own CSV file, one row per timestep")

There is no possibility to "add the time" to the VectorPostprocessor file directly.  Each column in the VectorPostprocessor file is all _at the same time_.... it's a whole set of information from one timestep.

This capability is writing an auxiliary file that contains time / timestep information in it to go along with the VectorPostprocessor output files.

Just making sure we're all on the same page here.